ThirdPartyNotices.txt

```diff
@@ -3350,6 +3350,39 @@ SOFTWARE.
 ===================================ExcelDataReader (END)
 
 -------------------------------END OF THIRD-PARTY NOTICES-------------------------------------------
+
```

Comment:
does a link to the chromium repo also need to be added in the first section of this file? Like when mxgraph was added in this PR https://github.com/microsoft/azuredatastudio/commit/f1371747198a662f7dfa258eeee04bc6aec40589

Reply:
Possibly? In the case of mxGraph we actually use that code directly to render query execution plans. In the case of chromium, it's used indirectly since electron uses it and ADS doesn't.

Reply:
@kisantia, it looks like the chromium license and source is already linked in our [cgmanifest.json](https://github.com/microsoft/azuredatastudio/blob/4ed287ac88fb16df4de0e2e87b9432c2d8628551/cgmanifest.json#L7)